src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -30,7 +34,7 @@ public class AddressBookParser {
      */
     private static final Pattern BASIC_COMMAND_FORMAT = Pattern.compile("(?<commandWord>\\S+)(?<arguments>.*)");
     private static final Pattern GET_COMMAND_FORMAT = Pattern
-            .compile("(?<commandWord>[get\\s]+)(?<prefix>[/a-z\\s]{4})(?<arguments>[a-z\\s].*)");
+            .compile("(?<commandWord>[get\\s]+)(?<prefix>[/a-z]+)(?<arguments>[a-zA-Z0-9\\s]*)");
```

Comment:
Good job changing the regex so that now `arguments` can take in numerical values and `prefix` is not limited to 4 characters only.